server/methods/core/shop.js

```diff
@@ -259,6 +259,7 @@ Meteor.methods({
       }
 
       Collections.Accounts.update(user._id, { $set: { "profile.currency": profileCurrency } });
+      Hooks.Events.run("afterAccountsUpdate", user._id, user._id);
```

Comment:
You could replace these two lines with your new method that already calls the hook

```js
Meteor.call("accounts/setProfileCurrency", profileCurrency);
```